sound/soc/sof/ipc4-topology.c

```diff
@@ -1970,6 +1971,47 @@ static int sof_ipc4_tear_down_all_pipelines(struct snd_sof_dev *sdev, bool verif
 	return 0;
 }
 
+static int sof_ipc4_dai_load(struct snd_sof_dev *sdev, struct snd_sof_pcm *spcm, int dir)
+{
+	struct sof_ipc4_timestamp_info *time_info;
+	u32 abi_version;
+	u32 abi_offset;
+
+	abi_offset = offsetof(struct sof_ipc4_fw_registers, abi_ver);
+	sof_mailbox_read(sdev, sdev->fw_info_box.offset + abi_offset, &abi_version,
+			 sizeof(abi_version));
+
+	if (abi_version != SOF_IPC4_FW_REGS_ABI_VER) {
```

Comment:
usually we avoid exact tests for the ABI checks, but use <= or some sort of inequality to avoid changing the code later if someone bumps the ABI in backwards compatible ways.

Reply:
sure, thanks